infra/dash/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.8.10-slim-buster
```

Comment:
this should be reviewed by @Dragos0000

Reply:
the given image was chosen as there is no special official image for the dash